### PR TITLE
fixed bug that sometimes makes wengy return only the first match

### PR DESCRIPTION
**Pull request by ingenium21**

### CODE REVIEW

**Comment by ingenium21** [comment]
This is to resolve Issue #64

**Comment by AndyReifman** [comment]
Can you please share the output from the command.

**Comment by ingenium21** [comment]
Testing:
ran bot locally in my private server
tested !ping
Wenger returned: "Pong"
tested !next

Wenger returned: 
Next Game
Next match is West Ham United (A) in 3 days, 19 hours, 21 minutes

tested !fixtures 
Wenger returned:

Next 3 Fixtures
West Ham United (A) - Premier League
16:30 - May 1
Arsenal (H) - Premier League
14:00 - May 8
Tottenham Hotspur (A) - Premier League
19:45 - May 12

tested !fixtures 10
Wenger returned: 

Next 7 Fixtures
West Ham United (A) - Premier League
16:30 - May 1
Arsenal (H) - Premier League
14:00 - May 8
Tottenham Hotspur (A) - Premier League
19:45 - May 12
Newcastle United (A) - Premier League
20:00 - May 16
Everton (H) - Premier League
16:00 - May 22
Arsenal (H) - Florida Cup Series
TBD - Jul 20
Arsenal (H) - Florida Cup
TBD - Jul 23

**Comment by ingenium21** [comment]
image

File: cogs/fixtures.py [hunk 1]
```diff
@@ -209,8 +209,19 @@ def findFixtures(matches, number):
...
-        location = match.find("div",{"class","location-icon"})['title']
+        try:
+            team = match.find("span",{"class","team-crest__name-value"}).text
+        except AttributeError:
+            team = match.find("div",{"class","team-crest__name-value"}).text
+        try:
+            location = match.find("div",{"class","location-icon"})['title']
+        except TypeError:
+            teams = match.findAll("div",{"class","fixture-match__team"})
+            homeAway = getLocation(teams)
```
Comment (ash663) [inline cogs/fixtures.py:220]: Does `getLocation` already exist?

Comment (ingenium21) [reply cogs/fixtures.py:220]: yes

**Review by ash663 — APPROVED** [review]

**Review by AndyReifman — APPROVED** [review]

**Pull request merged**